src/helpers/time-to-first-byte-analyser.helper.spec.js

```diff
@@ -0,0 +1,22 @@
+import { analyser } from './time-to-first-byte-analyser.helper';
+import { expect } from 'chai';
+
+const requests = [ { ttfb: 0, url: 'http://localhost:8080/' },
+  { ttfb: 2, url: 'http://localhost:8080/assets/kittens.jpg' },
+  { ttfb: 1, url: 'http://localhost:8080/favicon.ico' },
+  { ttfb: 2, url: 'http://localhost:8080/tabular-data' },
+  { ttfb: 1, url: 'http://localhost:8080/assets/kittens.jpg' } ];
+
+describe('Time to first byte analyser', () => {
+  it('returns found slow request', () => {
+    const maxTiming = 0; // equal to the lowest request's TTFB value
+
+    expect(analyser.analyseTTFB(requests, maxTiming)).to.deep.include({ ttfb: 2, url: 'http://localhost:8080/assets/kittens.jpg' });
```

Comment:
The same as in parser. You're testing the method that is not going to be used from the outside. `analyseTTFB` is just a private method and it should not be tested. There is one public method here and it's the one that is going to be used all the time - `checkTiming`.

What should be changed is that you should return a `create` method, not an instance. This way you'll be able to mock a parser.